src/datanode/src/tests/instance_test.rs

```diff
@@ -392,6 +393,58 @@ async fn test_rename_table() {
 "
     .to_string();
     check_output_stream(output, expected).await;
+
+    execute_sql_in_db(&instance, "select * from demo", "db").await;
```

Comment:
If any assertion in this test fails, the `#[should_panic]` attribute would still consider this test is passed. You could add a `try_execute_sql_in_db()` function and ensure its result is Err in this case.
```
async fn execute_sql_in_db() -> Output {
    try_execute_sql_in_db().await.unwrap()
}

async fn try_execute_sql_in_db() -> Result<Output> {}
```

Reply:
A nice catch, thanks